docs/src/components/home/Hero.tsx

```diff
@@ -110,12 +110,12 @@ export default function Hero() {
       left={
         <Box sx={{ textAlign: { xs: 'center', md: 'left' } }}>
           <Typography variant="h1" sx={{ my: 2, maxWidth: 500 }}>
-            The React <GradientText>UI&nbsp;library</GradientText> you always wanted
+            <GradientText>Move faster</GradientText> with intuitive React UI tools
           </Typography>
           <Typography color="text.secondary" sx={{ mb: 3, maxWidth: 500 }}>
-            MUI provides a robust, customizable, and accessible library of foundational and advanced
-            components, enabling you to build your design system and develop React applications
-            faster.
+            MUI offers a comprehensive suite of UI tools to help you ship new features faster. Start
+            with one of our fully-loaded component libraries like Material UI, or bring your own
```

Comment:
> one of our fully-loaded component libraries

We only have one 😁

Reply:
The idea there was to leave it open for the products to come that will be "parallel" to Material UI, while still mentioning Material UI by name for those who would be looking for it. (Also, would it be incorrect to describe _MUI X_ as a "component library"? Am I missing something about the usage of "library" in this context?)

Alternatively, we could do something like this: 

```suggestion
            with Material UI, our fully-loaded component library, or bring your own
```

Reply:
What do you think about this @mbrookes? Should we make it technically accurate today, or should we leave it open-ended for future products?

Reply:
Better to reflect reality I think.